.goreleaser.yml

```diff
@@ -0,0 +1,70 @@
+before:
+  hooks:
+    - go mod download
+    - go mod verify
+builds:
+  - main: ./cmd/mattermost-mattermod
```

Comment:
We don't necessarily need to release this binary since we already publish docker image.